src/ansys/fluent/core/launcher/launcher.py

```diff
@@ -101,6 +101,7 @@ def launch_fluent(
     topy: Optional[Union[str, list]] = None,
     start_watchdog: Optional[bool] = None,
     scheduler_options: Optional[dict] = None,
+    remote_file_handler: Optional[Any] = PimFileTransferService(),
```

Comment:
@hpohekar You must default it to None and instantiate it inside the body of the contructor if it's None.

Reply:
@hpohekar the issue specifies `file_transfer_service`

Reply:
> @hpohekar You must default it to None and instantiate it inside the body of the contructor if it's None.

Done. Thank you.

Reply:
> @hpohekar the issue specifies `file_transfer_service`

@seanpearsonuk Yesterday we decided that we will add `remote_file_handler` parameter in `launch_fluent` and `launch_remote_fluent` by keeping that point in mind I added `remote_file_handler` everywhere but after seeing your comment I updated it to `file_transfer_service` now.